test/int/epoch/process_epoch_fn.zig

```diff
@@ -0,0 +1,66 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+const TestCachedBeaconStateAllForks = @import("test_utils").TestCachedBeaconStateAllForks;
+const state_transition = @import("state_transition");
+const ReusedEpochTransitionCache = state_transition.ReusedEpochTransitionCache;
+const EpochTransitionCache = state_transition.EpochTransitionCache;
+
+pub fn getTestProcessFn(process_epoch_fn: anytype, no_alloc: bool, no_err_return: bool, no_void_return: bool) type {
```

Comment:
```suggestion

const TestOpt  = struct {
    alloc: bool = false,
    err_return: bool = false,
    void_return: bool = false,
};

pub fn TestRunner(process_epoch_fn: anytype, opt: TestOpt) type {
```

Two things here:
1. IMO, let's not overload the terminology `get` - this is doing a bunch of logic to decide how and what to run. Also nit: PascalCase for ` TestRunner` since it's a type.

2. Consider not using the negative terms `no_alloc`, `no_void_return`, `no_err_return` and instead use `alloc`, `void_return`, `err_return`. These are already boolean values - we can avoid double negatives here